parts/k8s/kuberneteswindowssetup.ps1

```diff
@@ -413,6 +413,11 @@ try
         Write-Log "Disable Internet Explorer compat mode and set homepage"
         Set-Explorer
 
+        # if multple LB policies are included for same endpoint then HNS hangs.
+        # this fix forces an error  
+        Write-Host "Enable a HNS fix in 2021-2C+"
+        reg add HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\hns\State /v HNSControlFlag /t REG_DWORD /d 1
+

```

Comment:
Will this ever be set by default in some future OS updates or will we always need to set this? 
Is this only needed on Windows Server 2019 or all Windows Server OS versions?

Reply:
@daschott do you have insight here?